libr/config/config.c

```diff
@@ -397,7 +398,7 @@ R_API RConfigNode* r_config_set(RConfig *cfg, const char *name, const char *valu
 	RConfigNode *node = NULL;
 	char *ov = NULL;
 	ut64 oi;
-
+	cfg->is_dirty = true;
```

Comment:
move below, that set cant be done before the asser preconditions

Reply:
I moved it down abit not sure if you wanted it to be there